### PR TITLE
test: Custom query system test

### PR DESCRIPTION
**Pull request by rishabkhawad**

### CODE REVIEW

File: tests/system/data_sources/test_bigquery.py [hunk 1]
```diff
@@ -989,3 +989,125 @@ def test_bigquery_row():
...
+                "target_column": "object_id",
+                "field_alias": "object_id",
+                "cast": None,
+            }
+        ],
+    }
+
+    client = data_validation.DataValidation(SAMPLE_CUSTOMQUERY_CONFIG)
+    result_df = client.execute()
+    print(result_df)
```
Comment (pshrivastava27) [inline tests/system/data_sources/test_bigquery.py]: can we get rid of this print statement?

Comment (rishabkhawad) [reply tests/system/data_sources/test_bigquery.py]: thanks for pointing out, print statement is now removed.

**Review by Raniksingh — APPROVED** [review]

**Pull request merged**